base/strings/substring.jl

```diff
@@ -47,6 +47,10 @@ end
 SubString(s::AbstractString) = SubString(s, 1, lastindex(s))
 SubString{T}(s::T) where {T<:AbstractString} = SubString{T}(s, 1, lastindex(s))
 
+@propagate_inbounds view(s::AbstractString, r) = SubString(s, r)
```

Comment:
With this general method, `view("asd", 2)` will return `"sd"` (or am I missing something?), whereas `getindex("asd", 2)` returns `'s'`, so it seems that this method should be split into more specialized arguments.

Reply:
I always forget that 1 argument substring is a thing. Will have to do that